[2024-01-16T06:34:12.601+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: jobinfo_slack.closeDeadline_sql_and_return_result manual__2024-01-16T06:34:05.874869+00:00 [queued]>
[2024-01-16T06:34:12.613+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: jobinfo_slack.closeDeadline_sql_and_return_result manual__2024-01-16T06:34:05.874869+00:00 [queued]>
[2024-01-16T06:34:12.614+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-01-16T06:34:12.615+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 2
[2024-01-16T06:34:12.615+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-01-16T06:34:12.642+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): closeDeadline_sql_and_return_result> on 2024-01-16 06:34:05.874869+00:00
[2024-01-16T06:34:12.647+0000] {standard_task_runner.py:55} INFO - Started process 64052 to run task
[2024-01-16T06:34:12.650+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'jobinfo_slack', 'closeDeadline_sql_and_return_result', 'manual__2024-01-16T06:34:05.874869+00:00', '--job-id', '740', '--raw', '--subdir', 'DAGS_FOLDER/jobinfo_slack.py', '--cfg-path', '/tmp/tmps0iy3hlr']
[2024-01-16T06:34:12.650+0000] {standard_task_runner.py:83} INFO - Job 740: Subtask closeDeadline_sql_and_return_result
[2024-01-16T06:34:12.731+0000] {task_command.py:388} INFO - Running <TaskInstance: jobinfo_slack.closeDeadline_sql_and_return_result manual__2024-01-16T06:34:05.874869+00:00 [running]> on host 565f6bcc10b6
[2024-01-16T06:34:12.831+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=jobinfo_slack
AIRFLOW_CTX_TASK_ID=closeDeadline_sql_and_return_result
AIRFLOW_CTX_EXECUTION_DATE=2024-01-16T06:34:05.874869+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-01-16T06:34:05.874869+00:00
[2024-01-16T06:34:12.852+0000] {base.py:73} INFO - Using connection ID 'rds_conn' for task execution.
[2024-01-16T06:34:12.942+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/jobinfo_slack.py", line 117, in closeDeadline_sql_and_return_result
    cursor.execute(sql_query)
  File "/home/airflow/.local/lib/python3.7/site-packages/MySQLdb/cursors.py", line 206, in execute
    res = self._query(query)
  File "/home/airflow/.local/lib/python3.7/site-packages/MySQLdb/cursors.py", line 319, in _query
    db.query(q)
  File "/home/airflow/.local/lib/python3.7/site-packages/MySQLdb/connections.py", line 254, in query
    _mysql.connection.query(self, query)
MySQLdb.OperationalError: (1054, "Unknown column 'mydb.jobinfo_jobplanet.company' in 'order clause'")
[2024-01-16T06:34:12.949+0000] {taskinstance.py:1323} INFO - Marking task as UP_FOR_RETRY. dag_id=jobinfo_slack, task_id=closeDeadline_sql_and_return_result, execution_date=20240116T063405, start_date=20240116T063412, end_date=20240116T063412
[2024-01-16T06:34:12.967+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 740 for task closeDeadline_sql_and_return_result ((1054, "Unknown column 'mydb.jobinfo_jobplanet.company' in 'order clause'"); 64052)
[2024-01-16T06:34:12.986+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2024-01-16T06:34:13.010+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
